### ISSUE TITLE
Resume podcast where it left off

### ISSUE THREAD

**Issue by dallanmc** [lucasasselli/garmin-podcasts#2]
Love the app (I left a review and donated). 
I'd also like to offer any programming help if you need it*. 

Anyway, I was wondering if it was possible to persist the last podcast played and the time in the podcast where it left off. If i restart the watch for example or if I stop listening to a podcast and try to resume later, then I've completely lost my place in the podcast that I was listening to. 

So is it possible to save the place in the podcast?

Also, the above might be helped by an actual "Now Playing" menu option in the the main menu which would bring you to the media player screen. Currently there's no intuitive way to get to the media player screen.

*you'd mentioned about the json limitations - I could perhaps provide an aws lambda function which would return a subset of only the necessary information.

**Comment by lucasasselli** [comment]
Hi Dallan,

I saw your comment and donation! Much appreciated! 👍 I'm taking some time off development since my watch is out for repair, but I will try to implement the changes you suggested as soon as it's back.

The now playing menu is a wonderful idea: Garmin allows to get/set the playback position, maybe it could be saved when the playback is stoopped. I don't think however that Garmin offers a way to quickly jump to the player, but I might be mistaken.

For the json limitations, I was discussing with the guys at PodcastIndex: they said that they were interested in implementing some mechanism to request only a subset of fields in the response, but they haven't implement it yet.

**Comment by dallanmc** [comment]
Thanks for the response! 
I've downloaded your source to have a look at helping out. Can I ask: how do you run the app normally? - is it through the simulator/emulator, or do you run it on the device? If you run it on the device, do you create a prg file and manually copy it over in order to test or is there a way to debug the app on the device while it's running?

Thanks in advance!.

**Comment by lucasasselli** [comment]
Normally using the simulator, by this is always the source of a lot of
problems: the simulator behaves quite differently from the device.

To run on the device you have to manually copy the prg file: on device
debug is not possible.

Usually when a crash occurs if the app is compiled for debug, a dump is
produced.

**Comment by lucasasselli** [comment]
I added a mechanism to save the current playback position and restore it in version 2.0.2. The position is stored on when the playback is stopped/paused and restarted when the user opens the podcast app again.

It's still very limited, but it's something. For the "Now playing" shortcut, I tried to implement it, but unfortunately Garmin doesn't offer a shortcut to the media player. `Media.startPlayback()` jumps to it BUT also resets the audio playback, which is not ideal.
